### ISSUE TITLE
SuiteCRM containers use PHP handler that is unsupported by SuiteCRM

### ISSUE THREAD

**Issue by kale1d0code** [bitnami/containers#47308]
### Name and Version

bitnami/suitecrm:7.13.3 possibly others

### What architecture are you using?

None

### What steps will reproduce the bug?

Try sending an automated email via AOW_Workflow

### What is the expected behavior?

For the CRM system to successfully send emails automatically

### What do you see instead?

AOW_Workflow does not provide email_address to SugarPHPMailer correctly. as SugarPHPMailer was provided a string instead of an array, the PHP process crashes.

This will result in none of the automated messages being sent.
there are other issues where the process crashes before it can be marked as completed and therefore it is retried every minute

### Additional information

running your SuiteCRM instance in older versions of the bitnami container will not cause your SuiteCRM instance to crash.

despite what https://docs.suitecrm.com/admin/compatibility-matrix/ says
SuiteCRM does not correctly support PHP 8.0

**Comment by dgomezleon** [comment]
Hi @kale1d0code ,

We are working on updating PHP for SuiteCRM. As soon as done we will notify you here.

**Comment by joancafom** [comment]
Hi @kale1d0code, 

We have recently released a new version of SuiteCRM, `7.14.0`, that ships PHP version `8.1`. This version is also expected to be compatible with the product, as stated in the official compatibility matrix.

```
$ docker run --rm bitnami/suitecrm:7.14.0 php --version
...

PHP 8.1.23 (cli) (built: Sep 21 2023 17:16:44) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.23, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.23, Copyright (c), by Zend Technologies
```

Could you please check if this new release resolves the problem?

**Comment by kale1d0code** [comment]
Excellent I'll look to upgrading.
However I still think you shouldn't bundle PHP 8.0 with SuiteCRM 7.13.x
It doesn't support it
